svg/shapes/ellipse-04-ref.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100">
+  <ellipse cx="50" cy="50" rx="0" ry="0" fill="none" stroke="red" stroke-width="5"/>
```

Comment:
Seem like this ref could trivially be an "empty" SVG - otherwise there's a risk that '0' & 'auto' is mishandled, but the test still passes.